src/System.Net.Sockets/tests/FunctionalTests/SendReceive.cs

```diff
@@ -20,7 +20,10 @@ public SendReceive(ITestOutputHelper output)
             _log = output;
         }
 
-        private static void SendToRecvFrom_Datagram_UDP(IPAddress leftAddress, IPAddress rightAddress)
+        [OuterLoop] // TODO: Issue #11345
+        [Theory]
+        [MemberData(nameof(LoopbacksToSameLoopback))]
```

Comment:
Good question. Since they're the same input, no idea. I'll change it throughout.